### DISCUSSION TITLE
Thank you for v.5 what is `recoil-sync`

### DISCUSSION THREAD

**Discussion by exrhizo** [facebookexperimental/Recoil#1419] [Q&A] [answered]
Hi,
Noticed the end of the release notes there is a mention of `recoil-sync`
What is involved with it?

Thank you!

**Comment by drarmstr** [comment, marked as answer]
https://github.com/facebookexperimental/Recoil/pull/1462/files

**Reply by salvoravida** [reply]
Hi, great work!

@drarmstr  should I use this official recoil-sync for ReduxBridge use case?

https://github.com/salvoravida/recoil-toolkit#electron-reduxbridge

**Comment by exrhizo** [comment]
Is there a since of timing of when recoil-sync is going to be released?

This interface feels on point for something I am planning building.

Maybe y'all might think this is overkill, but I want to use `react-query` as the main fetching interface and have that feed into recoil selectors. That way I don't have to implement caching but then I can do things that interact with the rest of my recoil logic.

**Reply by drarmstr** [reply]
Just busy getting the 0.6 release out now.  But, the implementation for the initial Recoil Sync library release is basically done, just need to finish the packaging and related infra.  The code is here if you want to play with it now: https://github.com/facebookexperimental/Recoil/tree/main/packages/recoil-sync
